src/components/data-exploration/generic-gene-table/ComponentActions.jsx

```diff
@@ -48,7 +48,7 @@ const ComponentActions = (props) => {
     </Menu>
   );
 
-  if (selectedGenes.length === 0) {
+  if (!selectedGenes?.length > 0) {
```

Comment:
This will break if `selectedGenes` is undefined. Have you checked if the code works when `selectedGenes` is undefined? Is it possible for it to be undefined?

Reply:
There is no case in the UI where the selected genes is `undefined`. I added this code in response to some tests that errors out. It seems I can fix it in the tests. I'll remove this, but add a check in`GeneSelectionMenu` just to ensure that `selectedGenes` is defined in these components (children of `GeneSelectionMenu`).

Reply:
I decided not to add the checks,